src/sagemaker_training/mpi.py

```diff
@@ -305,23 +305,24 @@ def run(self, wait=True, capture_error=False):
         logging_config.log_script_invocation(cmd, self._env_vars)
 
         training_env = environment.Environment()
-        exception_classes = get_modelparallel_exception_classes()
+        exception_classes = []
+        exception_classes += process.get_debugger_exception_classes()
+        if training_env.is_modelparallel_enabled:
+            exception_classes += get_modelparallel_exception_classes()
+        if exception_classes == [errors.ExecuteUserScriptError, errors.ExecuteUserScriptError]:
```

Comment:
what if the default exception name changed in future? what if there is another library with similar logic added? we can do a deduplication here instead of comparing to `[errors.ExecuteUserScriptError, errors.ExecuteUserScriptError]`

Reply:
list -> set -> list

Reply:
thanks! modified